src/api/UserApi.ts

```diff
@@ -119,7 +119,7 @@ function userDeviceAdd(
  */
 function userKeyList(sign: MessageSignature, userList: string[]) {
     return {
-        url: `users?id=${userList.join(",")}`,
+        url: `users?id=${userList.map(encodeURIComponent).join(",")}`,
```

Comment:
Don't the commas need to get encoded as well?

Reply:
It won't technically need to be until we go to v2 signature, but we might as well do it now IMO.

Reply:
Done